### DISCUSSION TITLE
Transaction

### DISCUSSION THREAD

**Discussion by valerii52** [Chia-Network/chia-blockchain#17963] [Support]
On May 1, at 23:01, I created a transaction to transfer 56 XCH coins to my OKX exchange account. The transaction was confirmed at block 5298616 memos 0c1ac0efa227a68e3583a6f64213acfe0076ad8133cc8af1e9d8b050b7db600f . The coins have not yet arrived in my account. Please indicate the reason.

**Comment by BrandtH22** [comment]
Hey @valerii52 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

By chance did you send this transaction with the clawback feature?

If so, OKX does not support the clawback claims so you will need to use the sending wallet to clawback the funds then send it to the OKX wallet without the clawback feature.

**Reply by valerii52** [reply]
Да, действительно, я отправил монеты с функцией clawback. Но сейчас у меня не получается вернуть их.

**Reply by valerii52** [reply]
Видимо, функция clawback не отвечает потому, что была установлена всего на 5 минут.

**Reply by valerii52** [reply]
Why do not you answer?

**Comment by valerii52** [comment]
I hope that the support service will still solve my problem and tell me how to get my coins back?

**Comment by BrandtH22** [comment]
Hey @valerii52 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

You will need to use the clawback feature to clawback the funds and then resend them without the clawback feature (OKX does not support this feature). 

To do so:
- on the transaction screen click the clawback option on the transaction

If this is not available then:
- in the settings / advanced click Resync|
- this will close the GUI and upon reopening will resync the wallet
- Once the wallet is resynced click the clawback option

If it is still not available then:
- stop chia
- rename the wallet db at `~/.chia/mainnet/wallet/db/` (delete any -shm or -wal files and rename the wallet db)
- start chia (the wallet will resync from scratch)
- Once the wallet is resynced click the clawback option

Lastly if it is still not available then we will want to run cli commands:
- using powershell (windows) or terminal (mac/ linux)
- Get the clawback transaction id by running `chia wallet get_transactions --clawback`
- Force the claim by running (replace TRANSACTION_ID with the transaction id from the previous step):
Windows: `chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'`
Mac/Linux: `chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`

NOTE: if you get errors running the above command, manually type in the command rather than pasting.

Lastly, please keep in mind that we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

**Comment by valerii52** [comment]
Hello again! I try one of your advices below...

"If it is still not available then:

stop chia
rename the wallet db at ~/.chia/mainnet/wallet/db/ (delete any -shm or -wal files and rename the wallet db)
start chia (the wallet will resync from scratch)
Once the wallet is resynced click the clawback option"

What does it means "rename wallet db"? What new name i should use?
I rename like this "~/.chia/mainnet/wallet/db1/" and after this all transactions disappeared. After multiple resyncs some of them appear again but clawback transaction still not available.
Help plz

**Reply by BrandtH22** [reply]
Hey @valerii52 I would then proceed to the final step which is a workaround to claim clawbacks using the cli commands (either terminal or windows powershell). 

As a reminder, we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

**Reply by BrandtH22** [reply]
Lastly if it is still not available then we will want to run cli commands:

    using powershell (windows) or terminal (mac/ linux)
    Get the clawback transaction id by running chia wallet get_transactions --clawback
    Force the claim by running (replace TRANSACTION_ID with the transaction id from the previous step):
    Windows: chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'
    Mac/Linux: chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'

NOTE: if you get errors running the above command, manually type in the command rather than pasting.

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
